### PR TITLE
add video url to faq

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr adds a video url to also explain how to solve the job memory limit failure in job run and development

internal slack: https://dbt-labs.slack.com/archives/C05CJT5MJF2/p1722309301858119?thread_ts=1721059396.203019&cid=C05CJT5MJF2

### CODE REVIEW

File: website/docs/faqs/Troubleshooting/job-memory-limits.md [hunk 1]
```diff
@@ -17,11 +17,10 @@ Some common reasons for higher memory usage are:
 
 There are various reasons why you could be experiencing this error. We recommend you review your data models to see if there are any opportunities to optimize or refactor them. For example, you can try to reduce the number of columns being selected, use `group` or `where` clauses to filter data early in the query, or use `limit` clauses to reduce the amount of data being processed.
 
-<!-- add url link when available
-:::tip Check out video example
-As an additional resource, check out this example video add URL here, which demonstrates how to refactor the sample code by reducing the number of columns returned. 
+:::tip Video example
+As an additional resource, check out [this example video](https://www.youtube.com/watch?v=sTqzNaFXiZ8), which demonstrates how to refactor the sample code by reducing the number of columns returned.
```
Comment (jeremyyeo) [inline website/docs/faqs/Troubleshooting/job-memory-limits.md]: > columns returned

rows returned

File: website/docs/faqs/Troubleshooting/job-memory-limits.md [hunk 2]
```diff
@@ -17,11 +17,10 @@ Some common reasons for higher memory usage are:
 
 There are various reasons why you could be experiencing this error. We recommend you review your data models to see if there are any opportunities to optimize or refactor them. For example, you can try to reduce the number of columns being selected, use `group` or `where` clauses to filter data early in the query, or use `limit` clauses to reduce the amount of data being processed.
```
Comment (jeremyyeo) [inline website/docs/faqs/Troubleshooting/job-memory-limits.md]: > There are various reasons why you could be experiencing this error. We recommend you review your data models to see if there are any opportunities to optimize or refactor them. For example, you can try to reduce the number of columns being selected, use `group` or `where` clauses to filter data early in the query, or use `limit` clauses to reduce the amount of data being processed.

There are various reasons why you could be experiencing this error but they are mostly the outcome of retrieving too much data back into dbt (via `run_query()` operations or similar macros, or even using database/schemas that have lots of other non-dbt related tables/views). You can try to reduce the amount of data / number of rows retrieved back into dbt by refactoring the SQL in your `run_query()` operation using `group`, `where` or `limit` clauses. Additionally, you can also use a database/schema with fewer non-dbt related tables/views.

**Review by jeremyyeo — CHANGES_REQUESTED** [review]

File: website/docs/faqs/Troubleshooting/job-memory-limits.md (hunk 1 above)
Comment (mirnawong1) [inline website/docs/faqs/Troubleshooting/job-memory-limits.md]:
```suggestion
As an additional resource, check out [this example video](https://www.youtube.com/watch?v=sTqzNaFXiZ8), which demonstrates how to refactor the sample code by reducing the number of rows returned. 
```

File: website/docs/faqs/Troubleshooting/job-memory-limits.md (hunk 2 above)
Comment (mirnawong1) [inline website/docs/faqs/Troubleshooting/job-memory-limits.md]:
```suggestion
There are various reasons why you could be experiencing this error but they are mostly the outcome of retrieving too much data back into dbt. For example, using the `run_query()` operations or similar macros, or even using database/schemas that have a lot of other non-dbt related tables/views. Try to reduce the amount of data / number of rows retrieved back into dbt by refactoring the SQL in your `run_query()` operation using `group`, `where`, or `limit` clauses. Additionally, you can also use a database/schema with fewer non-dbt related tables/views.


```

**Comment by mirnawong1** [comment]
thanks @jeremyyeo ! can you please have another look?

**Review by jeremyyeo — APPROVED** [review]

**Pull request merged**